internal/resolvers/proc/hns.go

```diff
@@ -21,25 +23,31 @@ type HNSProc struct {
 	Verbose          bool
 	procStarted      bool
 	height           uint64
-	lastHeightUpdate time.Time
-	synced           bool
+	progress         float64
 	retryCount       int
 	lastRetry        time.Time
+	hsClient         *dns.Client
 	sync.RWMutex
 }
 
-func NewHNSProc(procPath string, rootAddr, recursiveAddr string) (*HNSProc, error) {
-	args := []string{"--ns-host", rootAddr, "--rs-host", recursiveAddr, "--pool-size", "4"}
+func NewHNSProc(procPath string, rootAddr, recursiveAddr string, configPath string) (*HNSProc, error) {
+	args := []string{"--ns-host", rootAddr, "--rs-host", recursiveAddr, "--pool-size", "4", "-x", configPath}
```

Comment:
Mine as well add `-t` as well to start from the hard coded checkpoint at height 30000 ;-)

Reply:
I misread this line from the PR and thought that `-t` would be ignored:
>`prefix` will override `checkpoint` if both are used.

Turns out it does help starting from 136k if there's no written file. Added `-t`, thanks